plugins/tekton/src/components/Icons/LinkToSbomIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+import { SVGIconProps } from '@patternfly/react-icons/dist/esm/createIcon';
+import classNames from 'classnames';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    icon: {
+      fill: 'var(--pf-v5-global--Color--100)',
+    },
+    disabledButton: {
+      fill: theme.palette.grey[600],
+    },
+  }),
+);
+
+const LinkToSBom: React.FC<SVGIconProps> = (props): React.ReactElement => {
```

Comment:
It'd be good to try and keep this component name consistent with the filename.  Since it's an icon I'd suggest calling the component `LinkToSBomIcon`.  What props does this support?  Or is the props argument there for convenience?  It'd be better to have well defined properties for this component.

Reply:
Oh, I see it'd be `SVGIconProps`, missed that...

Reply:
Updated the export name.